wynnset/tapestry-wp
# 540 - Change external link to open in new tab by default

Closes #540 

Note that all tapestries created prior to this PR will not have the new tab selected by default (e.g. when editing). The warning message however will show regardless.

**Testing**

* Create a new node
* Change the node type to url embed
  * The "Open in a new window" radio button should be selected by default
* Press the "Embed in Tapestry" option
  * A warning message should appear telling the user that links may not work correctly.
